src/scss/components/file/mixins/hxFileControl.scss

```diff
@@ -0,0 +1,66 @@
+@import "vars";
```

Comment:
@lalithkarikelli, please note the changes in this file.  Thanks.

Reply:
Changes have been done, please review.